oshu_manor.lic

```diff
@@ -10,6 +10,17 @@ class Oshu
   include DRCI
 
   def initialize
+    arg_definitions = [
+      [
+        { name: 'critter', options: %w(worms kartais), description: 'What critter you hunting?' }
```

Comment:
Another option would be:
```
arg_definitions = [
  [
    { name: 'worm', regex: /^worm/i, description: 'Hunt grave worms' }
  ],
  [
    { name: 'kartais', regex: /^kartais/i, description: 'Hunt seordhevor kartais' }
  ]
]
```

Do you think the output from that is any more user-friendly?